build.ss

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env gxi
 ;;; -*- Gerbil -*-
-(import :std/build-script)
+(import :std/build-script
+        :std/misc/process)
+
+(invoke "gxtags" ["."])
```

Comment:
this is generally unnecessary with gerbil pkg; you can just tag the package once you've linked it.